packages/common/src/store/pages/chat/sagas.ts

```diff
@@ -519,7 +528,21 @@ function* doCreateChatBlast(action: ReturnType<typeof createChatBlast>) {
         audienceContentType
       }
     })
-    yield* call(track, make({ eventName: Name.CREATE_CHAT_FAILURE }))
+
+    const encodedContentId = audienceContentId
+      ? encodeHashId(audienceContentId)
+      : undefined
+
+    yield* call(
+      track,
+      make({
+        eventName: Name.CREATE_CHAT_BLAST_FAILURE,
+        audience,
+        audienceContentType,
+        audienceContentId: encodedContentId,
```

Comment:
super nit, but curious whether we want this encoded or not

Reply:
yeah I wasn't sure either 🤔